.circleci/config.yml

```diff
@@ -21,12 +27,29 @@ jobs:
       - run:
           name: "Build Snap"
           command: snapcraft
+      - run:
+          name: Generate tarball
+          command: |
+            VERSION=$(bash circleci.sh --version)
```

Comment:
As with the Git tag created later on, can we prefix all versions with the letter 'v'? The character doesn't have to been in the envar itself but in strings when it's concatenated in them.